packages/core/src/navigation/navigation-start.element.ts

```diff
@@ -120,36 +123,70 @@ export class CdsNavigationStart extends LitElement implements FocusableItem {
 
   private handleStartButtonText() {
     spanWrapper(this.childNodes);
+
     // get the projected text now wrapped in a span and add the sr attribute.
     this.querySelector('span')?.setAttribute(NAVIGATION_TEXT_WRAPPER, '');
   }
 
+  private renderRootStart() {
```

Comment:
For a more accessible navigation, if there is a root start element, it is tab-able and has its own onKey handlers for expand/collapse behavior.